code/modules/client/preferences_savefile.dm

```diff
@@ -356,7 +356,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["feature_ipc_antenna"]				>> features["ipc_antenna"]
 	S["feature_ipc_chassis"]				>> features["ipc_chassis"]
 	S["feature_insect_type"]				>> features["insect_type"]
-	if(!CONFIG_GET(flag/join_with_mutant_humans)  && !species_id != "felinid") // felinids arent mutant humans anymore i guess
+	if(!CONFIG_GET(flag/join_with_mutant_humans))
 		features["tail_human"] = "none"
```

Comment:
Why are you removing the ability for felinids to select human tails if you are only disabling the config option?

Reply:
Because there are multiple human tails now, with only one being a cat tail. The only thing a felinid could do is select tail or no tail. Only difference here is now they tail always.  Its either this, or they get monky tails (and perhaps clockwork as well once you merge that damn fix).